include/swift/AST/Decl.h

```diff
@@ -595,7 +595,10 @@ class alignas(1 << DeclAlignInBits) Decl : public ASTAllocated<Decl> {
     HasLazyConformances : 1,
 
     /// Whether this nominal type is having its semantic members resolved.
-    IsComputingSemanticMembers : 1
+    IsComputingSemanticMembers : 1,
+
+    /// Whether ~BitwiseCopyable appeared in the type's inheritance list.
+    SuppressedBitwiseCopyable : 1
```

Comment:
I'd prefer not to add a specific bit for BitwiseCopyable here, because there are other protocols for which the `~` will make sense now and in the near future (e.g., `Sendable`, `Equatable`, and `Hashable`). Can we instead store a set of suppressed protocols elsewhere?

Reply:
Changed this bit to indicate whether suppressed conformances had been computed.